chainer/functions/math/sum.py

```diff
@@ -6,7 +6,9 @@
 class Sum(function.Function):
     """Sum of array elements over a given axis."""
 
-    def __init__(self, axis=None):
+    keepdims = False
```

Comment:
Is this necessary here? `self.keepdims` seems to have some value unless error is raised.